CIScripts/Build/BuildFunctions.ps1

```diff
@@ -203,7 +203,45 @@ function Invoke-AgentBuild {
                      -MSIPath $agentMSI
 
     $Job.Step("Copying artifacts to $OutputPath", {
-        Copy-Item $agentMSI $OutputPath -Recurse -Container
+        Copy-Item $agentMSI $OutputPath -Recurse
+    })
+
+    $Job.PopStep()
+}
+
+function Invoke-NodemgrBuild {
+    Param ([Parameter(Mandatory = $true)] [string] $OutputPath,
+           [Parameter(Mandatory = $true)] [string] $LogsPath,
+           [Parameter(Mandatory = $false)] [string] $BuildMode = "debug")
+
+    $Job.PushStep("Nodemgr build")
+
+    $Job.Step("Building nodemgr", {
+        $Components = @(
+            "database:node_mgr",
+            "build/debug/sandesh/common/dist",
```

Comment:
s/build/`$BuildMode`/
Also, maybe create a PR that adds an alias to scons.
Perhaps an alias to build the contrail-nodemgr with all its dependencies?